### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by ntfshard**

### CODE REVIEW

**Comment by ntfshard** [comment]
make test

**Comment by ntfshard** [comment]
ea33a6c1-79e4-f1a9-ba0e-8cdcd4b723a1

has dependency, 208 first

File: algorithms/kernel/k_nearest_neighbors/kdtree_knn_classification_predict_dense_default_batch_impl.i [hunk 1]
```diff
@@ -348,7 +348,6 @@ void KNNClassificationPredictKernel<algorithmFpType, defaultDense, cpu>::findNea
 
     DAAL_ALIGNAS(256) algorithmFpType distance[__KDTREE_LEAF_BUCKET_SIZE + 1];
     size_t start, end;
-    algorithmFpType dist, diff, val;
```
Comment (averbukh) [inline algorithms/kernel/k_nearest_neighbors/kdtree_knn_classification_predict_dense_default_batch_impl.i]: Double check that you do not affect performance here

File: examples/java/com/intel/daal/examples/compression/CompressionOnline.java [hunk 2]
```diff
@@ -155,7 +155,7 @@ private static void prepareMemory() throws java.io.FileNotFoundException, java.i
             currentBlock = Arrays.copyOfRange(sentDataStream, (int) startPosition,
                     (int) (startPosition + maxDataBlockSize));
             availableDataSize -= maxDataBlockSize;
-        } else if ((availableDataSize < maxDataBlockSize) && (availableDataSize > 0)) {
```
Comment (averbukh) [inline examples/java/com/intel/daal/examples/compression/CompressionOnline.java]: Non-equivalent changes

File: examples/java/com/intel/daal/examples/compression/CompressorExample.java [hunk 3]
```diff
@@ -145,7 +145,7 @@ private static void prepareMemory() throws java.io.FileNotFoundException, java.i
             currentBlock = Arrays.copyOfRange(sentDataStream, (int) startPosition,
                     (int) (startPosition + maxDataBlockSize));
             availableDataSize -= maxDataBlockSize;
-        } else if ((availableDataSize < maxDataBlockSize) && (availableDataSize > 0)) {
```
Comment (averbukh) [inline examples/java/com/intel/daal/examples/compression/CompressorExample.java]: Non-equivalent changes

File: examples/java/com/intel/daal/examples/compression/CompressionOnline.java (hunk 2 above)
Comment (ntfshard) [reply examples/java/com/intel/daal/examples/compression/CompressionOnline.java]: Looks legit 4 me

File: examples/java/com/intel/daal/examples/compression/CompressorExample.java (hunk 3 above)
Comment (ntfshard) [reply examples/java/com/intel/daal/examples/compression/CompressorExample.java]: Looks legit 4 me

File: algorithms/kernel/k_nearest_neighbors/kdtree_knn_classification_predict_dense_default_batch_impl.i (hunk 1 above)
Comment (ntfshard) [reply algorithms/kernel/k_nearest_neighbors/kdtree_knn_classification_predict_dense_default_batch_impl.i]: Ok

**Comment by ntfshard** [comment]
ea3618ce-ba92-f119-864b-8cdcd4b723a1

**Comment by ntfshard** [comment]
ea368d65-de08-f1db-8a6e-8cdcd4b723a1

File: examples/java/com/intel/daal/examples/compression/CompressionOnline.java (hunk 2 above)
Comment (averbukh) [reply examples/java/com/intel/daal/examples/compression/CompressionOnline.java]: OK

File: examples/java/com/intel/daal/examples/compression/CompressorExample.java (hunk 3 above)
Comment (averbukh) [reply examples/java/com/intel/daal/examples/compression/CompressorExample.java]: OK

**Pull request merged**